.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     steps:
```

Comment:
What do you think about making two different jobs here so we can run in parallel? It also makes it easier to clean up when we stop supporting Xcode 11. So I'm imagining that we change `build-macos` to `build-macos-on-xcode-11` and then add a `build-macos-on-xcode-12`.

Reply:
That makes total sense to me. I'll update.